tillerinobot/src/main/java/tillerino/tillerinobot/lang/LanguageIdentifier.java

```diff
@@ -36,6 +36,7 @@
 	Català(Catalan.class),
 	Slovenščina(Slovenian.class),
 	Schwiizerdütsch(Swissgerman.class),
+	Slovenčina(Slovak.class),
```

Comment:
Is "Slovenčina" fine here?

Reply:
Well, if you want it in alphabetical order, it's Slovenčina, Slovenščina, Schwiizerdutsch, if u don't want it like that, this is okay :D

Reply:
Okay, thx